### DISCUSSION TITLE
add "back" and "forward"  on toolbar

### DISCUSSION THREAD

**Discussion by rockjablew** [sumatrapdfreader/sumatrapdf#4575] [Ideas]
add "back" and "forward"  on toolbar

**Comment by GitHubRulesOK** [comment]
unsure which one you want already have 2
 but add your own `B` & `F` or other word
![image](https://github.com/user-attachments/assets/443a1740-cdaf-495e-9d11-628259984223)



CmdNavigateBack | Alt + Left, Backspace | Navigate Back
-- | -- | --
CmdNavigateForward | Alt + Right, Shift + Backspace | Navigate Forward



3.6 PRE_RELEASE
```
Shortcuts [
	[
		Cmd = CmdNavigateBack
		Key = b
		ToolbarText = <Bk
	]
	[
		Cmd = CmdNavigateForward
		Key = f
		ToolbarText = Fw>
	]
```

**Reply by rockjablew** [reply]
yes,i want this

**Reply by rockjablew** [reply]
i used Portable version

**Reply by GitHubRulesOK** [reply]
portable 32bit/64bit/arm or installed does not matter it works in all 6 x 3.6 versions.
